README.md

```diff
@@ -137,6 +139,17 @@ curl -X 'POST'   'http://localhost:8080/assistant/chat/streaming' -H 'Content-Ty
 
 The `assistantName` can be swapped out for other assistants inside of the table above, but the other assistants will required a connection to a weaviate db with the correct indexes. The App Of Apps repository contains a [validation script](https://github.com/redhat-composer-ai/appOfApps/blob/main/data-ingestion/weaviate/validation.sh) that can be used to show which indexes currently exist.
 
+## Local Curl with File Upload
+
+To send a local curl request with an uploaded file, the following command may be used:
+
+```sh
+curl 'http://localhost:8080/assistant/chat/streaming' -F 'jsonRequest={
```

Comment:
Nice! At some point I want to clean up all the sections in this README and get all this info into our centralized docs.

(No changes needed here just wanted to say thanks for documenting)